operator/v1/types_console.go

```diff
@@ -113,6 +113,19 @@ type ConsoleCustomization struct {
 	// +kubebuilder:validation:Optional
 	// +optional
 	DeveloperCatalog DeveloperConsoleCatalogCustomization `json:"developerCatalog,omitempty"`
+	// projectAccess defines configuration options for defining project access roles.
+	// +kubebuilder:validation:Optional
+	// +optional
+	ProjectAccess ProjectAccess `json:"projectAccess,omitempty"`
+}
+
+// ProjectAccess contains options for project access roles
+type ProjectAccess struct {
+	// availableClusterRoles specifies the list of ClusterRoles that are assignable to users
+	// through the project access tab.
+	// +kubebuilder:validation:Optional
+	// +optional
+	AvailableClusterRoles []string `json:"availableClusterRoles"`
```

Comment:
do we need empty list here ? if not then lets add 
```suggestion
	AvailableClusterRoles []string `json:"availableClusterRoles, omitempty"`
```

Reply:
Updated.